src/query/storages/index/src/bloom.rs

```diff
@@ -248,7 +254,15 @@ impl BlockFilter {
         };
 
         let (filter, _size) = Xor8Filter::from_bytes(filter_bytes)?;
-        if filter.contains(&target) {
+
+        let contains = if self.version == 2 {
```

Comment:
can we change the constant `2` to an enum value BlockBloomFilterIndexVersion::V2?

Reply:
This way, it will be easier for us to consider it when we refactor it in the future.

Reply:
ok